.github/workflows/gh-pages.yml

```diff
@@ -22,17 +22,30 @@ jobs:
           node-version: 20
           registry-url: "https://registry.npmjs.org"
 
-      - name: Run typedoc
+      - name: Install dependencies and build project
         run: |
-          npm install 
+          npm install
           npm run bootstrap
           npm run build
+
+      - name: Run typedoc
+        run: |
           npm run typedoc
 
-      - name: Fix typedoc links
+      - name: Run Solidity Docgen
+        run: |
+          npm run docs
+        working-directory: contracts
+
+      - name: Install website dependencies
         run: |
           npm install
+        working-directory: website
+
+      - name: Fix typedoc and solidity docgen links
+        run: |
           npm run setup-typedoc
+          npm run setup-soliditydocs
```

Comment:
Could we introduce a script to simplify the website generation process? Running a single command like `./build-website.sh` would make the build steps much more efficient.

Reply:
Done